src/plugins/shared_dictionaries_plugin/shared_dictionaries_plugin_settings.cpp

```diff
@@ -0,0 +1,13 @@
+#include "shared_dictionaries_plugin_settings.hpp"
+
+namespace opossum {
+
+const std::string& JaccardIndexThresholdSetting::description() const {
+  static const auto description = std::string{"Threshold for the similarity metric between dictionaries for merging"};
+  return description;
+}
+const std::string& JaccardIndexThresholdSetting::display_name() const { return _display_name; }
+const std::string& JaccardIndexThresholdSetting::get() { return _value; }
+void JaccardIndexThresholdSetting::set(const std::string& value) { _value = value; }
```

Comment:
Maybe, the setter could check whether the new value falls into the pre-defined range of [0.0, 1.0].

Reply:
Fixed in https://github.com/hyrise/hyrise/pull/2394/commits/7f88317d0867f9bfd92688f5061daea73a39be35